src/resources/assets/js/components/filteringTab.vue

```diff
@@ -1,97 +1,62 @@
-        @click.once="loadShapes"
-        v-model="selectedAnnotationShape"
-      >
-        <option
-          v-for="(annotation_type_name, annotation_type_id) in this.shapes"
-          :value="annotation_type_id"
-          v-text="annotation_type_name"
-        ></option>
-      </select>
-    </div>
-    <div class="list-group filter-list-group">
-      <label>Users with Annotations</label>
-      <select
-        class="form-control"
-        @change="filterAnnotation"
-        @click.once="loadUsers"
-        v-model="selectedAnnotationUser"
-      >
-        <option
-          v-for="(annotation_type_name, annotation_type_id) in this.possibleUsers"
-          :value="annotation_type_id"
-          v-text="annotation_type_name"
-        ></option>
-      </select>
+  <div class="filtering-tab"
+>
+  <label>Annotations where: </label>
+  <ul class="list-group">
+      <li class="list-group-item" v-for="filter, k in activeFilters">
+          <span>{{ filter.name }}</span>
+          <button @click.once="removeFilter(k)" type="button"
+          class="btn btn-default  fa fa-window-close" ></button>
+      </li>
+  </ul>
+  <form class="form clearfix" v-on:submit.prevent>
+    <div class="filter-form__selects">
+        <annotation-filter @add-filter="addNewFilter">
+        </annotation-filter>
     </div>
+    </form>
+
   </div>
 </template>
 <script>
+import AnnotationFilter from '../components/annotationFilter.vue'
+
 export default {
-  props: {
-    annotationShapesLoader: {
-      type: Function,
-      required: true,
-    },
-    possibleUsersLoader: {
-      type: Function,
-      required: true,
-    },
+  components: {
+    AnnotationFilter
   },
   data() {
     return {
-      selectedAnnotationShape: null,
-      selectedAnnotationUser: null,
-      shapes: {
-        0: null
-      },
-      possibleUsers: {
-        0: null
-      },
+      activeFilters: []
     };
   },
   methods: {
-    filterAnnotation() {
-      //TODO: add more filters here. See https://github.com/biigle/largo/issues/66
-      let selectedFilters = {
-        shape_id: this.selectedAnnotationShape,
-        user_id: this.selectedAnnotationUser,
-      };
-      //Filter out null filters, this can cause bad requests to be sent
-      Object.keys(selectedFilters).forEach(
-        (k) => {
-          if (selectedFilters[k] == null || selectedFilters[k] == 0) {
-            delete selectedFilters[k]
-          }
-        }
-      );
-      this.$emit("handle-selected-filters", selectedFilters);
-    },
-
-    async loadShapes() {
-      if (Object.keys(this.shapes).length == 1) {
-        let loadedShapes = await this.annotationShapesLoader();
-        this.shapes = { ...this.shapes, ...loadedShapes }
+    removeFilter(key){
+      this.activeFilters.splice(key, 1)
+      //If its the first filter to be removed, remove the 'And' or 'Or'
+      if (key == 0){
+        this.activeFilters[0].name = this.activeFilters[0].name.substring(
```

Comment:
When I remove the last filter rule, I get `this.activeFilters[0] is undefined` in the console.